### PR TITLE
Replace ... with :

### PR DESCRIPTION
**Pull request by maccarth32**
Issue 1000
I really like using : a lot better

### CODE REVIEW

**Comment by pagrubel** [comment]
Thank you for the pull request.

**Review by pagrubel — APPROVED** [review]

**Review by bernhold — COMMENTED** [review]
Hi, thanks for your pull request!

In the future, I would encourage you to provide a  title and description/comment for the PR that better describes the change you made.  I realize you used what was in our instructions, but the *change* we suggested was consistent with the comment we suggested (though in retrospect, I realize we could probably do a better job in the instructions).

If this were a real software project, I, as the maintainer, might ask you to revise the title and description (first comment) before I would accept it.  I might want to do that to ensure that in the future, I can look through (or search) a list of PRs and have a better chance at identifying the one I was looking for.  You have the ability to change the title by clicking the Edit button the far right of the title.  You can edit the comment by clicking on the ellipsis ("...") menu on the right side of the particular comment.

Alternatively, as maintainer, I might go ahead and make the changes I felt were necessary before merging you PR. 

In this case, I see that my fellow maintainer Pat got in ahead of me and has merged your contribution.  Not all maintainers are alike :-)

I hope you've found this useful.  We're happy to continue providing feedback on issues and PRs, or feel free to email us at the address we provided yesterday.

Thanks!

**Comment by pagrubel** [comment]
And this is a good way to communicate to me did it in too much of a hurry. The PR could have been reverted by David if it really needed to go back to review.

**Pull request merged**
